app/views/components/_user_nav_drawer.html.erb

```diff
@@ -14,3 +14,12 @@
       </li>
   </ul>
 </nav>
+
+<!-- <a class="nav-sign-out" href="/micropurchase/sign-in/">
```

Comment:
Normally I would say yes, but this commented out code is going to be added soon, so it is essentially staged here.